troubleshooting.md

```diff
@@ -0,0 +1,550 @@
+
+# Troubleshooting Flow 
+
+Prior to troubleshooting your environment, it is critical to understand the overall flow of the IPI on BM installation. The diagram below is a troubleshooting flowchart providing a step-by-step breakdown of where to start debugging your environment.
+
+![Flow-Diagram](https://github.com/DirectedSoul1/baremetal-deploy/blob/master/images/OpenShift%20Baremetal%20Deployment%20Troubleshooting%20Flow.png)
+
+## Install-config 
+
+The install-config.yaml represents all the nodes that will be part of your OpenShift cluster and the necessary options consisting of but not limited to `apiVersion`, `baseDomain`, `networking`, `VIPS`, `imageContentSources` etc. 
+
+During the deployment of your OpenShift cluster, if errors occur early on due to the `install-config.yaml`, confirm the following
+
+* Use the guidelines as mentioned in the [Yaml-tips](https://www.redhat.com/sysadmin/yaml-tips)
+* Verifying syntax is correct within your YAML using [syntax-check](http://www.yamllint.com/)
+* Verify the RHCOS qemu images are properly defined and accessible via the URL provided in the `install-config.yaml`.
+
+# Bootstrap VM issues 
+
+The bootstrap node is a virtual machine that is triggered by the OpenShift installer that handles the provisioning of the nodes that will be part of the OpenShift cluster.
+
+Once the deployment is triggered (after waiting for 10 to 15 minutes) we can see if the Bootstrap VM is operational. We can verify the bootstrap VM is operational via the `virsh` command.
+
+```
+[kni@provisioner]$ sudo virsh list
+ Id    Name                           State
+----------------------------------------------------
+ 12    kni7-xf6fq-bootstrap           running
+```
+
+If no bootstrap VM is running after waiting 10-15 minutes, it is important to troubleshoot as to why it was not created. Possible issues to verify are:
+
+* Verify `libvirtd` is running on the system
+
+```
+$ systemctl status libvirtd
+● libvirtd.service - Virtualization daemon
+   Loaded: loaded (/usr/lib/systemd/system/libvirtd.service; enabled; vendor preset: enabled)
+   Active: active (running) since Tue 2020-03-03 21:21:07 UTC; 3 weeks 5 days ago
+     Docs: man:libvirtd(8)
+           https://libvirt.org
+ Main PID: 9850 (libvirtd)
+    Tasks: 20 (limit: 32768)
+   Memory: 74.8M
+   CGroup: /system.slice/libvirtd.service
+           ├─ 9850 /usr/sbin/libvirtd
+```
+
+
+If the bootstrap VM is indeed operational, we want to verify if we can log into it. The IP of this VM can be obtained by issuing `sudo virsh net-dhcp-leases` Typically, the private IP of the bootstrap VM is `172.22.0.2`. To access the VM use the  `core` user for logging in. 
+We can find the IP address of the bootstrap VM using the `virsh console` command as shown. 
```

Comment:
extra line break to have it on different paragraph

Reply:
Add a line break as suggested.